pygmt/figure.py

```diff
@@ -263,6 +263,7 @@ def savefig(
         - JPEG (``.jpg`` or ``.jpeg``)
         - PDF (``.pdf``)
         - BMP (``.bmp``)
+        - PPM (``.ppm``)
```

Comment:
Maybe it make sense to also sort this list alphabetically?

Reply:
Thanks. Done in 473bef8f4354b181c0a3386b26d5b48989bc7854.